automatic/chromium/tools/chocolateyInstall.ps1

```diff
@@ -15,7 +15,7 @@ if (Test-Path $Chromium) {
 
 $packageArgs = @{
   packageName   = 'chromium'
-  file          = "$toolsdir\chromium_x32.exe"
+  url           = 'https://chromium_x32.exe'
```

Comment:
Since you are conditionally installing 32bit and 64bit where one is embedded, and the other is a remote binary you need to make additional changes as well.

When 32bit is attempted to be installed, this needs to be done through the `Install-ChocolateyPackage` cmdlet, not the `Install-ChocolateyInstallPackage` which the 64bit version needs.

There is also a missing `checksum` and `checksumType` property for the 32bit executable.